src/styles/globals.css

```diff
@@ -168,4 +195,80 @@
     display: flex;
     justify-content: center;
     border: solid black 3px;
+    border-bottom: 0;
+}
+
+.wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
+.navbar {
+    display: flex;
+    flex-direction: column;
+}
+
+.main {
+    display: flex;
+    justify-content: center;
+}
+
+.bodywrapper {
+    display: flex;
+    height: calc(100vh - 100px);
+    flex-direction: row-reverse;
+}
+
+.footer-pos {
+    display: flex;
+    justify-content: center;
+}
+
+.body {
+    width: 100%;
+    height: calc(100vh - 100px);
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    flex-direction: column;
+}
+
+.pokeball {
+    display: flex;
+    align-items: flex-start;
+    height: 100%;
+    background: url(/pixelpokeball.png) 0 0;
+    aspect-ratio: 75 / 100;
+}
+
+.pokeball:hover {
+    /* background: url(/pixelpokeball.png) -148px 0; */
+    animation-name: pokeballopening;
+    animation-duration: 1s;
+    animation-fill-mode: forwards;
+}
+
+.themebutton {
+    margin: 10px;
+}
+
+@keyframes pokeballopening {
+    0% {
+        background: url(/pixelpokeball.png) 0 0;
+    }
+    20% {
```

Comment:
Allgemein sind die Steps in deiner Animation etwas Strange. Effektiv beginnt deine Animation erst bei 20% und endet bei 60.001%. 0-20% und 60.001-100% machen nichts. Das führt dazu, dass deine Animation effektiv viel weniger lang dauert als deine oben angegebene Zeit.

Ändere die Frames gerne dahingehend, dass bei 0% die erste Änderung passiert, und bei 100% die letzte.